yoga/Yoga.c

```diff
@@ -379,6 +379,14 @@ static void YGNodeMarkDirtyInternal(const YGNodeRef node) {
   }
 }
 
+static void YGNodeMarkDirtyRecursive(const YGNodeRef node) {
```

Comment:
Sorry for the churn on this. I should have looked closer. `YGNodeMarkDirtyInternal` is already recursive so this actually doesn't make much sense ^^ You previous name is better suited 👍